src/main/java/com/example/gradecalculator/controller/UserSubjectController.java

```diff
@@ -73,28 +83,29 @@ public ResponseEntity<Map<String, Object>> saveUserSubject(
         }
     }
 
-    @GetMapping("userSubject/selected")
+    @GetMapping("/userSubject/selected")
```

Comment:
every method mapping starts with "/userSubject" -> use an additional mapping on class level 
for example see GradesController